types/lhr.d.ts

```diff
@@ -27,6 +27,13 @@ declare global {
       benchmarkIndex: number;
     }
 
+    export interface StackPacks {
+      id: string;
+      icon: string;
+      detectedLibraries: string[];
+      advice: Record<string, string>;
```

Comment:
kinda want to call this "descriptions" or something like ****Descriptions. Just so we are always using the same term.